dessia_common/utils/helpers.py

```diff
@@ -72,3 +77,14 @@ def get_python_class_from_class_name(full_class_name: str):
     # Storing in cache
     _PYTHON_CLASS_CACHE[full_class_name] = class_
     return class_
+
+
+def get_attributes_type(class_: Generic[T], name: str):
+    """ Get type of attribute name of class."""
+    var_type = None
```

Comment:
Avoid prefixing variable names with element like `var_`.

If you need to escape reserved word `type`, add an "escaping underscore" at the end like : `type_`

In addition to that, avoid variable name abbreviation, ex : `parameters` instead of `param` or `arguments` instead of `arg`